From f009608eba8455e28e81c2c54091d0b3984a9922 Mon Sep 17 00:00:00 2001
From: Daniel Borkmann <dborkman@redhat.com>
Date: Tue, 22 Jan 2013 11:31:42 +0100
Subject: [PATCH 35/53] test_autoclose: remove unneeded variable
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fixes:

  test_autoclose.c: In function ‘main’:
  test_autoclose.c:68:13: warning: variable ‘bytes_sent’ set but not used [-Wunused-but-set-variable]

We don't need it here since otherwise the test will fail anyway.

$ ./test_autoclose
test_autoclose.c  0 INFO : Waiting for the associations to close automatically in 5 secs
test_autoclose.c  1 PASS : Autoclose of associations

Signed-off-by: Daniel Borkmann <dborkman@redhat.com>
---
 src/func_tests/test_autoclose.c | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/src/func_tests/test_autoclose.c b/src/func_tests/test_autoclose.c
index 902375c..b5368e7 100644
--- a/src/func_tests/test_autoclose.c
+++ b/src/func_tests/test_autoclose.c
@@ -65,7 +65,7 @@ main(int argc, char *argv[])
 	sockaddr_storage_t loop1, loop2;
 	struct msghdr inmessage, outmessage;
 	struct iovec iov, out_iov;
-	int error, bytes_sent;
+	int error;
 	char *big_buffer;
 	char *message = "hello, world!\n";
 	uint32_t autoclose;
@@ -113,7 +113,8 @@ main(int argc, char *argv[])
 	outmessage.msg_iovlen = 1;
 	outmessage.msg_iov->iov_base = message;
 	outmessage.msg_iov->iov_len = strlen(message) + 1;
-	bytes_sent = test_sendmsg(sk1, &outmessage, 0, strlen(message)+1);
+
+	test_sendmsg(sk1, &outmessage, 0, strlen(message)+1);
 
 	/* Initialize inmessage for all receives. */
 	big_buffer = test_malloc(REALLY_BIG);
-- 
1.7.11.7

